### DISCUSSION TITLE
Welcome to NASA-Earth-Data Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [opengeos/NASA-Earth-Data#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BriannaLind** [comment]
My name is Bri!  I like to look at Earth (but mostly termite mounds in African savannas) from space to understand how ecosystems function and might to adapt to change. I also work at[ NASA's Land Process DAAC](https://github.com/nasa/LPDAAC-Data-Resources) (aka data library).
